docs/pages/machine-id/workload-identity/best-practices.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Best Practices for Teleport Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+This page covers common questions and best practices for using Teleport's
+Workload Identity feature in production.
+
+## Structuring SPIFFE IDs
+
+The SPIFFE ID is a flexible identifier for workloads, and potentially even
+users. It is ultimately up to your organization to decide how to structure
+your SPIFFE ID namespace.
+
+There's a few common strategies to structuring SPIFFE IDs. Generally, though, a
+hierarchical structure is used, with the root of the hierarchy being the most
+general and the deepest part being the most specific. This allows rules to be
+created which allow access to a group of workloads that share common parts
+of the hierarchy.
+
+### Logical structure
+
+One strategy is to structure SPIFFE IDs based on the logical function of a
+workload. For example, you may have a service named `processor` that falls
+within a group of `payments` services. You could give this a SPIFFE ID of
+`spiffe://example.teleport.sh/production/payments/processor`. You may then
+decide that all `payments` services should be able to access each other and
+create a rule that allows access to any SPIFFE ID that starts with
+`spiffe://example.teleport.sh/production/payments`.
+
+### Physical structure
+
+Another strategy is to structure the SPIFFE IDs using a more "physical" location
+of a workload. This may still include elements that are "virtual". For example,
+you may have a workload running on a VM on a host in a datacenter in London.
+You could give this a SPIFFE ID of
+`spiffe://example.teleport.sh/europe/uk/london/hypervisor-001/vm-a3847f`.
+This is useful in a different way to the logical structure, it'd be ideal in a
+case where you want to restrict a workload to only be able to access other
+workloads which are physically proximate, such as a cache. You may say that
+the cache located in London will only accept connections from workloads with a
+SPIFFE ID that starts with `spiffe://example.teleport.sh/europe/uk`.
```

Comment:
this is a helpful section

Reply:
I'm sure we'll learn more as time goes on by this is the extent of my experience with it - I think we can provide a lot of value in generic SPIFFE information (that makes us look like an authoritative figure) that isn't currently easily accessible on the net - plus maybe some SEO benefits.